src/test/java/org/embulk/input/zendesk/services/TestZendeskUserEventService.java

```diff
@@ -0,0 +1,158 @@
+package org.embulk.input.zendesk.services;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.embulk.EmbulkTestRuntime;
+import org.embulk.input.zendesk.RecordImporter;
+import org.embulk.input.zendesk.ZendeskInputPlugin.PluginTask;
+import org.embulk.input.zendesk.clients.ZendeskRestClient;
+import org.embulk.input.zendesk.utils.ZendeskTestHelper;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class TestZendeskUserEventService
```

Comment:
I have few concerns here:
* UT only tests the function the class exposes, we don't expose guess, run or preview in `ZendeskUserEventService`, I understand that conventionally `execute` is used in `preview` and `run`, `getData` is used in `guess` phase, but we're designing the new method and we supposed that it could be invoked in any place.
* I'm not 100% sure but is there different test cases naming style in the same plugin, if you look at what you have done before: https://github.com/treasure-data/embulk-input-zendesk/blob/716c8a581a7d0310229d5f8a3687334bf097c306/src/test/java/org/embulk/input/zendesk/TestZendeskInputPlugin.java#L242-L250 and what you just wrote: `testUrlForUserEvent`.

And that concern not only apply for this `UserEventService` but also apply for others.

Reply:
fixed